LayoutTests/fast/repaint/highlight-paint-after-range-change-expected.txt

```diff
@@ -0,0 +1,5 @@
+Highlight this first and then this.
```

Comment:
Would be good to add an indication of how to read the test results. There's been cases where rebaselining these ended up in regressing the test.

Something like: "Test passes if [...]"

Reply:
I think the `title` element in the test has a pretty reasonable description, but I suppose it could be moved into the body instead using either `js-test.js` + `description(…)`, or as a separate element (such that it shows up in the `-expected.txt`).

Reply:
I think a description of which repaint rects (numbers) are expected is generally useful. It's possible for someone to rebaseline such a test with the wrong repaints happening. It has happened to me, and I've seen rebaselines of dumps where the rebaseline actually turned out to be wrong.